notebooks/scripts/hamming_distance_from_fasta.py

```diff
@@ -19,6 +19,7 @@
 
     parser.add_argument("--alignment", required=True, help="aligned FASTA file of diseases")
     parser.add_argument("--output", required=True, help="path for the csv output")
+    parser.add_argument("--indel-distance", action="store_true", help="add indel distances to genetic distances")
```

Comment:
I tested this script with the MERS alignment first using the default without indels:

```
python3 notebooks/scripts/hamming_distance_from_fasta.py \
  --alignment mers-nextstrain/results/aligned.fasta \
  --output test.csv
```

This command ran as expected. Then, I ran the command again but with indel distances:

```
python3 notebooks/scripts/hamming_distance_from_fasta.py \
  --alignment mers-nextstrain/results/aligned.fasta \
  --indel-distance \
  --output test_with_indels.csv
```

This command failed with the following error:

```python
Traceback (most recent call last):
  File "notebooks/scripts/hamming_distance_from_fasta.py", line 35, in <module>
    args.indel_distance
  File "/Users/jhuddlesfredhutch.org/projects/cartography/notebooks/scripts/Helpers.py", line 112, in get_hamming_distances
    np_genomes = np.array(list(re.sub("-", "0", genomes[i])))
TypeError: 'odict_values' object is not subscriptable
```

It also looks like the doctests for the hamming distance function aren't passing:

```bash
python3 -m doctest notebooks/scripts/Helpers.py
```

Gives:

```python
**********************************************************************
File "notebooks/scripts/Helpers.py", line 77, in Helpers.get_hamming_distances
Failed example:
    get_hamming_distances(genomes)
Expected:
    [1, 2, 2]
Got:
    [0, 1, 1]
**********************************************************************
File "notebooks/scripts/Helpers.py", line 80, in Helpers.get_hamming_distances
Failed example:
    get_hamming_distances(genomes)
Expected:
    [1, 1, 1]
Got:
    [0, 0, 0]
**********************************************************************
File "notebooks/scripts/Helpers.py", line 83, in Helpers.get_hamming_distances
Failed example:
    get_hamming_distances(genomes)
Expected:
    [3, 2, 4]
Got:
    [2, 0, 2]
**********************************************************************
1 items had failures:
   3 of   8 in Helpers.get_hamming_distances
***Test Failed*** 3 failures.
```

Can you work through the changes needed to get these tests passing before we do more here, @nandsra21?